pkg/security/secl/model/model.go

```diff
@@ -361,11 +361,16 @@ func (f *FileFields) GetInUpperLayer() bool {
 // FileEvent is the common file event type
 type FileEvent struct {
```

Comment:
Should we create setters for PathnameStr + IsPathnameStrResolved and BasenameStr + IsBasenameStrResolved ? (to make sure we don't modify one without changing the other)

Reply:
Thanks for the suggestion. I updated the PR